recipes/qt/all/conanfile.py

```diff
@@ -0,0 +1,687 @@
+import os
+import shutil
+import itertools
+import glob
+
+import configparser
+from conans import ConanFile, tools, __version__ as conan_version, RunEnvironment
+from conans.errors import ConanInvalidConfiguration
+from conans.model import Generator
+from conans.tools import Version
+
+
+class qt(Generator):
+    @property
+    def filename(self):
+        return "qt.conf"
+
+    @property
+    def content(self):
+        return "[Paths]\nPrefix = %s\n" % self.conanfile.deps_cpp_info["qt"].rootpath.replace("\\", "/")
+
+
+def _getsubmodules():
+    config = configparser.ConfigParser()
+    config.read('qtmodules.conf')
+    res = {}
+    assert config.sections()
+    for s in config.sections():
+        section = str(s)
+        assert section.startswith("submodule ")
+        assert section.count('"') == 2
+        modulename = section[section.find('"') + 1: section.rfind('"')]
+        status = str(config.get(section, "status"))
+        if status != "obsolete" and status != "ignore":
+            res[modulename] = {"status": status,
+                               "path": str(config.get(section, "path")), "depends": []}
+            if config.has_option(section, "depends"):
+                res[modulename]["depends"] = [str(i) for i in config.get(section, "depends").split()]
+    return res
+
+
+class QtConan(ConanFile):
+
+    _submodules = _getsubmodules()
+
+    generators = "pkg_config"
+    name = "qt"
+    description = "Qt is a cross-platform framework for graphical user interfaces."
+    topics = ("conan", "qt", "ui")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.qt.io"
+    license = "LGPL-3.0"
+    exports = ["qtmodules.conf", "patches/*.diff"]
+    settings = "os", "arch", "compiler", "build_type"
+
+    options = {
+        "shared": [True, False],
+        "commercial": [True, False],
+
+        "opengl": ["no", "es2", "desktop", "dynamic"],
+        "with_vulkan": [True, False],
+        "openssl": [True, False],
+        "with_pcre2": [True, False],
+        "with_glib": [True, False],
+        # "with_libiconv": [True, False],  # Qt tests failure "invalid conversion from const char** to char**"
```

Comment:
```suggestion
        # "with_libiconv": [True, False],  # XXX: Qt tests failure "invalid conversion from const char** to char**"
```

I would add a common tag to filter it

Reply:
would [QTBUG-84708](https://bugreports.qt.io/browse/QTBUG-84708) do it ?